ophyd/areadetector/trigger_mixins.py

```diff
@@ -30,11 +30,15 @@ class TriggerBase(BlueskyInterface):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         # settings
-        self.stage_sigs.update([(self.cam.acquire, 0),  # If acquiring, stop.
-                                (self.cam.image_mode, 1),  # 'Multiple' mode
-                                ])
+        # careful here: quadEM devices have areadetector components but,
+        # they have no 'cam' plugin. See QuadEM initializer.
+        if hasattr(self, 'cam'):
+            self.stage_sigs.update([(self.cam.acquire, 0),  # If acquiring, stop
+                                    (self.cam.image_mode, 1),  # 'Multiple' mode
+                                    ])
+            self._acquisition_signal = self.cam.acquire
```

Comment:
The only functionality the base class gives above `BlueskyInterface` is the two `stage_sigs`. These options offer little benefit compared to just updating `stage_sigs` manually:

```python
# attribute names by kwarg
TriggerBase(..., start_acquiring='cam.acquire', start_acquiring_value=0, 
            acquisition_mode='cam.image_mode', acquisition_mode_value=1)

# case where it has no acquisition mode signal for example
TriggerBase(..., start_acquiring=None, acquisition_mode=None)

# specifying just cam requires that 'image_mode' and 'acquire' are the same
TriggerBase(..., cam_attr='cam')
TriggerBase(..., cam_attr=None)
```

Do we need another layer on top of TriggerBase that expects a `cam_attr` (i.e., `ADTriggerBase(prefix, *, cam_attr='cam'`) such that `TriggerBase` can be reused for `cam`-less quadEM and others? Or should `TriggerBase` be exclusively areaDetector-with-cam?

cc @danielballan